### PR TITLE
feat(cts): add data source to get a list of CTS traces

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
feat(cts): add data source to get a list of CTS traces
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add data source to get a list of CTS traces
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
 make testacc TEST="./huaweicloud/services/acceptance/cts" TESTARGS="-run TestAccDataSourceCtsTraces_basic"
==> Checking that code complies with gofmt requirements...

TF_ACC=1 go test ./huaweicloud/services/acceptance/cts -v -run TestAccDataSourceCtsTraces_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceCtsTraces_basic
=== PAUSE TestAccDataSourceCtsTraces_basic
=== CONT  TestAccDataSourceCtsTraces_basic
--- PASS: TestAccDataSourceCtsTraces_basic (436.28s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cts       436.367s
```

### CODE REVIEW

File: docs/data-sources/cts_traces.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Use this data source to get the list of CTS traces.
+
+## Example Usage
+
+```hcl
+variable "from" {}
+variable "to" {}
+
+data "huaweicloud_cts_notifications" "test" {
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cts_traces.md]:
```suggestion
data "huaweicloud_cts_traces" "test" {
```

Comment (profoundwu) [reply docs/data-sources/cts_traces.md]: fixed

**Pull request merged**